### DISCUSSION TITLE
Welcome to notpixel_bot_advanced Discussions!

### DISCUSSION THREAD

**Discussion by WubbaLubbaDubDubDev** [WubbaLubbaDubDubDev/notpixel_bot_advanced#12] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by danazor** [comment]
The bot is very efficient in terms of points. Thank you for that. I have a few suggestions.

Can you simplify the outputs a bit more? I mean,

it also writes the name of the map in each repaint.

..... map.png
12 pts.
..... map.png
12 pts.
..
..
..

...map.png
... 12pts
... 12pts

I think it would be easier to follow if it was like that.

+ Is there a chance to show the number of repaints as well?

**Reply by WubbaLubbaDubDubDev** [reply]
Good idea about simplifying the output. As for the number of repaints, it might be more appropriate to display this parameter at the end of the drawing cycle. What do you think?

**Comment by danazor** [comment]
I agree. It can give 1 time template information at the beginning and show the total repaint score at the end. by the way, thanks for the improvements

**Reply by WubbaLubbaDubDubDev** [reply]
> I agree. It can give 1 time template information at the beginning and show the total repaint score at the end. by the way, thanks for the improvements

In the latest update, I removed the message about loading templates from the local cache. Now, the template information will only be displayed during subscription and on the first load. I also added the output for TotalRepaints after the drawing cycle concludes.

**Reply by WubbaLubbaDubDubDev** [reply]
I was thinking, maybe it would be better to add template information to each log entry with information about the painted pixel?
session | Painted on (x=499, y=597) with color #6D482F | templateID: 1075675229 | reward: 12.0
